src/scopyExceptionHandler.h

```diff
@@ -0,0 +1,27 @@
+#ifndef SCOPYEXCEPTIONHANDLER_H
+#define SCOPYEXCEPTIONHANDLER_H
+
+#include <QMessageBox>
+#include <QApplication>
+#include <config.h>
+#include <libm2k/m2kexceptions.hpp>
+#include <scopyApplication.hpp>
+
+using namespace adiscope;
+
+#define HANDLE_EXCEPTION(e) \
```

Comment:
Why a macro and not a simple function / static function in a class

Reply:
__FILE__ and __LINE__ macros will set file and line number @ the location where they are used. If you use a simple function, they will always have same file name and line. If you use a macro, __FILE__ and __LINE__ will be placed where HANDLE_EXCEPTION is called and replaced with the actual file and line number where handle exception is used